caiman/tests/test_sbx.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+import os
+
+import numpy as np
+import numpy.testing as npt
```

Comment:
Can we alphabetise these parts of the imports? (the caiman imports going later is fine